RoadMap_SwiftUI/RoadMap_SwiftUI/CartView.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  CartView.swift
+//  RoadMap_SwiftUI
+//
+//  Created by Артур Щукин on 20.01.2023.
+//
+
+import SwiftUI
+/// Экран корзины
+struct CartView: View {
+    
+    // MARK: - Public properties
+    var body: some View {
+        VStack(spacing: Constants.Numbers.vpnPlusVerticalSpacing) {
+            Text(Constants.Texts.inYourCartText)
+                .font(.largeTitle)
+                .fontWeight(.bold)
+            HStack {
+                Text(viewModel.selectedOptionText)
+                Text("\(viewModel.selectedOptionPrice)")
+            }
+            makeAnOrderButtonView
+        }
+    }
+    
+    // MARK: - Private properties
+    @EnvironmentObject private var viewModel: VPNViewModel
+    @State private var isCheckoutAlertShown = false
+    private var makeAnOrderButtonView: some View {
```

Comment:
```suggestion

    private var makeAnOrderButtonView: some View {
```
 отделяй враперы от свойств